src/components/v2/MyAccount/MyAccount.tsx

```diff
@@ -0,0 +1,106 @@
+import { ISliderProps, Slider } from '../Slider';
+import { useMyAccountStyles as useStyles } from './MyAccountStyles';
+import { Toggle } from '../Toggle';
+import { Icon } from '../Icon';
+import { Tooltip } from '../Tooltip';
+
+const myAccountResources = {
+  myAccountTitle: 'My account',
+  apyWithXvs: 'APY with XVS',
+  apyWithXvsTooltip: 'tooltip content',
+  netApy: 'Net APY',
+  dailyEarnings: 'Daily earnings',
+  supplyBalance: 'Supply balance',
+  borrowBalance: 'Borrow balance',
+  borrowLimit: 'Borrow limit:',
+  safeLimit: 'Your safe limit:',
+};
+
+interface IMyAccountProps {
+  netApy: string;
+  dailyEarnings: string;
+  supplyBalance: string;
+  borrowBalance: string;
+  borrowLimit: string;
+  borrowLimitSum: string;
+  safeLimit: string;
+  sliderProps: ISliderProps;
+}
+
+export const MyAccount = ({
+  netApy,
+  dailyEarnings,
+  supplyBalance,
+  borrowBalance,
+  borrowLimit,
+  borrowLimitSum,
+  safeLimit,
+  sliderProps,
+}: IMyAccountProps) => {
+  const [isToggleSwitched, setToggleSwitched] = useState(true);
+  const handleSwitch = (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => {
+    setToggleSwitched(checked);
+  };
+  const styles = useStyles();
+  return (
+    <Box css={styles.root} component={Paper}>
+      <div css={styles.row}>
+        {myAccountResources.myAccountTitle}
+        <span css={styles.apyWithXvs}>
+          <Tooltip css={styles.tooltip} title={myAccountResources.apyWithXvsTooltip}>
+            <span>
+              <Icon css={styles.iconInfo} name="info" />
+            </span>
+          </Tooltip>
+          {myAccountResources.apyWithXvs}
+          <Toggle cssStyles={styles.toggle} value={isToggleSwitched} onChange={handleSwitch} />
+        </span>
+      </div>
+      <div>
+        <p css={[styles.label, styles.netApyLabel]}>
+          {myAccountResources.netApy}
+          <Tooltip css={styles.tooltip} title={myAccountResources.apyWithXvsTooltip}>
+            <span>
+              <Icon css={styles.iconInfo} name="info" />
+            </span>
+          </Tooltip>
+        </p>
+        <span css={styles.netApy}>{netApy}</span>
+      </div>
+      <ul css={styles.list}>
+        <li css={styles.item}>
+          <p css={[styles.label, styles.labelListItem]}>{myAccountResources.dailyEarnings}</p>
+          {dailyEarnings}
+        </li>
+        <li css={styles.item}>
+          <p css={[styles.label, styles.labelListItem]}>{myAccountResources.supplyBalance}</p>
+          {supplyBalance}
+        </li>
+        <li css={styles.item}>
+          <p css={[styles.label, styles.labelListItem]}>{myAccountResources.borrowBalance}</p>
+          {borrowBalance}
+        </li>
+      </ul>
+      <div css={[styles.row, styles.borrowLimit]}>
+        <div>
+          <p css={[styles.label, styles.inline]}>{myAccountResources.borrowLimit}</p> {borrowLimit}
+        </div>
+        {borrowLimitSum}
+      </div>
+      <Slider {...sliderProps} />
```

Comment:
It seems the green bar inside the slider can be moved by clicking anywhere on the slider, but it should be fixed.

Reply:
added disabled prop